src/test/groovy/pl/nextleveldev/smart_route/ApplicationContextLoadTest.groovy

```diff
@@ -0,0 +1,23 @@
+package pl.nextleveldev.smart_route
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.context.ApplicationContext
+import spock.lang.Specification
+
+@SpringBootTest
+class ApplicationContextLoadTest extends Specification {
```

Comment:
Change the name to `ApplicationContextloadspec` to maintain compliance with Spock Conventions.

Spock Convention: Test classes in Spock traditionally have a Spec suffix (e.g. ApplicationContextLoadSpec), which is the framework's convention. This makes it easier to recognize that this is a test written with Spock.

Does the name change have an impact?

Changing the name to Spec will not affect the behavior of the tests, because Gradle and Spring Boot recognize tests based on the annotations (@SpringBootTest, @Test, etc.) and the directory structure.

Spock vs. JUnit: In JUnit, test classes often have a Test suffix, but in Spock the class name ends with Spec, which is more natural for the framework.